motionplan/plannerOptions.go

```diff
@@ -45,12 +44,13 @@ const (
 	defaultSmoothIter = 20
 
 	// names of constraints.
-	defaultLinearConstraintName        = "defaultLinearConstraint"
-	defaultPseudolinearConstraintName  = "defaultPseudolinearConstraint"
-	defaultOrientationConstraintName   = "defaultOrientationConstraint"
-	defaultObstacleConstraintName      = "defaultObstacleConstraint"
-	defaultSelfCollisionConstraintName = "defaultSelfCollisionConstraint"
-	defaultJointConstraint             = "defaultJointSwingConstraint"
+	defaultLinearConstraintName         = "defaultLinearConstraint"
+	defaultPseudolinearConstraintName   = "defaultPseudolinearConstraint"
+	defaultOrientationConstraintName    = "defaultOrientationConstraint"
+	defaultObstacleConstraintName       = "defaultObstacleConstraint"
+	defaultSelfCollisionConstraintName  = "defaultSelfCollisionConstraint"
+	defaultRobotCollisionConstraintName = "defaultRobotCollisionConstraint"
```

Comment:
I know I asked earlier what I should be calling this, but I was planning on leaving this as is for now, the reason being I know @biotinker is working on an overhaul of this whole system and there might be a better way to structure the name, error across the board for constraints.  Thoughts on this? Should I make the error overhaul its own ticket or do it here?

Reply:
+1 for adding a new ticket for the error overhaul. I think it's a good thing to do after @biotinker finishes up his work.